tests/devices.spec.ts

```diff
@@ -106,3 +110,42 @@ test("profile edit networks", async ({ page }) => {
 
   await deleteProfile(page, profile);
 });
+
+test("add, edit and remove an other device", async ({ page, lxdVersion }) => {
+  test.skip(
+    lxdVersion === "5.0-edge",
+    "Other device configuration not supported in lxd v5.0/edge",
+  );
+  const profile = randomProfileName();
+
+  //Add Other Device
+  await startProfileCreation(page, profile);
+  await page.getByText("Other", { exact: true }).click();
+  await page.getByRole("button", { name: "Attach custom device" }).click();
+  await page.getByLabel("Type").selectOption("USB");
+  await page.getByLabel("Bus Number").fill("123");
+  await page.getByLabel("Device Number").fill("12345");
+  await finishProfileCreation(page, profile);
+
+  //Edit and save Other Device
+  await visitProfileOtherDevices(page, profile);
+  await page.getByLabel("Device Number").fill("123456");
+  await saveProfile(page, profile, 1);
+
+  //Edit and save Other Device (Refresh)
```

Comment:
```suggestion
  // Refresh to check values after edit
```
...and maybe add a space also before the other comments?